mayastor/src/grpc/mayastor_grpc.rs

```diff
@@ -125,6 +125,23 @@ impl MayastorSvc {
     }
 }
 
+impl TryFrom<CreatePoolRequest> for PoolArgs {
+    type Error = LvsError;
```

Comment:
I don't think this should be a LvsError rather a gRPC Error?

Reply:
This call is made inside a spdk thread so the returned error should be LvsError 
https://github.com/openebs/mayastor/blob/067c7f4270835dd4c475be6de631d1dbc37fa5af/mayastor/src/grpc/mayastor_grpc.rs#L245
If we take grpcError we need to convert it again.

Also the PoolArgs struct is declared inside the lvs lib so taking LvsError